tools/ios/assemble_pod_package.py

```diff
@@ -81,16 +81,22 @@ def assemble_pod_package(
     pod_name = "onnxruntime-extensions-c"
     pod_summary = "ONNX Runtime Extensions C/C++ Pod"
     pod_description = "Pod containing pre and post processing custom ops for onnxruntime."
-    with open(framework_info_file, mode="r") as f:
-        framework_info = json.load(f)
-        ios_deployment_target = framework_info["apple_deployment_target"]
+    with open(xcframework_info_file, mode="r") as f:
+        xcframework_info = json.load(f)
+        ios_deployment_target = xcframework_info["iphonesimulator"]["apple_deployment_target"]
```

Comment:
If I was to do a build with just 'iphoneos' would this work? 

Could we be more generic/consistent and iterate the keys with a case for each instead of hardcoding one key here and processing the macos value separately?

e.g. maybe one case for key.startswith('iphone') to cover 'iphoneos' and 'iphonesimulator' (or a comment explaining why we only check iphonesimulator and not iphoneos) and one case with 'MacOSX' in key?

Reply:
yeah good point technically speaking, it's possible to only specify `--platform iphoneos arm64`for the build. and this won't work. though currently we have a default set where all archs are included for the case platform argument is not specified.

updated as suggested.

Reply:
yeah good point. it's possible to do a build with only specifying `--platform iphoneos` and this probably won't work.

updated as suggested.